Sherlock/pubsub.h

```diff
@@ -336,11 +336,12 @@ class PubSubHTTPEndpointImpl : public AbstractSubscriberObject {
       return ss::EntryResponse::Done;
     }
     if (serving_) {
-      if (us > to_timestamp_) {
+      // Stop serving if the limit on timestamp is exceeded.
+      if (to_timestamp_.count() && us > to_timestamp_) {
         return ss::EntryResponse::Done;
       }
       if (!params_.array && !params_.entries_only) {
-        http_response_(JSON<J>(ts_optidx_t(us)));
+        http_response_(JSON<J>(ts_optidx_t(us)) + "\n");
```

Comment:
Right.
BTW, `JSON(ts_optidx_t(timestamp))` produces strings like `{us:"<timestamp>", index:null}`. Is there a simple way to get rid of this `index:null` part?
And one more question: if `J` isn't `JSONFormat::Minimalistic`, `JSON<J>(ts_optidx_t(us, index))` and `JSON<J>(idxts_t(index, us))` produce different strings with different type_ids, right? If so, we need more "compatible" solution.